app/src/main/java/org/oppia/android/app/databinding/GuidelineBindingAdapters.java

```diff
@@ -16,4 +16,13 @@ public static void setGuidelinePercentage(@NonNull Guideline guideline, float pe
     params.guidePercent = percentage;
     guideline.setLayoutParams(params);
   }
+
+  /** Binding adapter for setting the `guideEnd` for a [Guideline]*/
+  @BindingAdapter("app:layout_constraintGuide_end")
+  public static void setConstraintGuidelineEnd(@NonNull Guideline guideline, float marginEnd) {
```

Comment:
Prefer ``guideEnd`` over ``marginEnd`` since that's what the value represents. Also, is the unit pixels? Prefer ``guideEndPx`` to ensure the unit is included in the name to avoid potential future misunderstandings.

Reply:
Yup, value is in pixels, therefor fixed it/